src/main/java/com/develhope/spring/vehicles/controller/VehicleController.java

```diff
@@ -16,12 +18,37 @@ public class VehicleController {
     @Autowired
     private VehicleService vehicleService;
 
-    @PostMapping
     public ResponseEntity<Vehicle> createVehicle(@RequestBody Vehicle vehicle) {
         Vehicle vehicleAdded = vehicleService.createVehicle(vehicle);
         return ResponseEntity.ok(vehicleAdded);
```

Comment:
visto che hai utilizzato ResponseEntity si potrebbero valutare anche i casi in cui lo status non è 200 ma un error, e quindi valutare sia la creazione riuscita che quella non riuscita.